t_stress/test_stress.py

```diff
@@ -457,6 +461,12 @@ def test(self):
         self.assertNotIn(" 0 2xx, ", client.response_msg)
 
 
+class H2LoadStressMTU80(H2LoadStress):
+    tempesta_to_client_mtu = 80
+    tempesta_to_server_mtu = 80
+    server_to_tempesta_mtu = 80
+
+
```

Comment:
@RomanBelozerov can you please add http/1 TLS stress as well with low MTU?

Reply:
done